tools/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF

```diff
@@ -13,12 +13,12 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.debug.core,
  org.eclipse.jface,
  biz.aQute.bndlib;bundle-version="3.3.0",
- org.eclipse.buildship.core;bundle-version="1.0.21",
- com.gradleware.tooling.model;bundle-version="0.16.0",
+ org.eclipse.buildship.core,
```

Comment:
Lets put a verison here, so we make sure we are using the 2.0.xx versions of the core plugin.